src/client.rs

```diff
@@ -28,6 +30,26 @@ impl Default for Config {
             host: "localhost".to_string(),
             port: "8126".to_string(),
             service: "".to_string(),
+            logging_config: None
+        }
+    }
+}
+
+#[derive(Clone, Debug)]
+pub struct LoggingConfig {
+    level: LogLevel,
+    time_format: String,
+    mod_filter: Vec<&'static str>,
+    body_filter: Vec<&'static str>,
+}
+
+impl Default for LoggingConfig {
+    fn default() -> Self {
+        LoggingConfig {
+            level: LogLevel::Info,
+            time_format: "%Y-%m-%d %H:%M:%S%z".to_string(),
```

Comment:
Should this be RFC3339 compatible? Should `z` be `Z`? Should there be a `T` between the date and time? I am not exactly sure how this is used but I feel that we should always use RFC3339 and let users do things they want to the time externally.

Reply:
I kept it this way because this is the way our ddog parser expects it.  If I change it, it could cause some major compatibility issues.

As for the space versus T, it's optional according to spec (as are upper vs. lower case z and t):

```
5.6. Internet Date/Time Format

   The following profile of ISO 8601 [ISO8601] dates SHOULD be used in
   new protocols on the Internet.  This is specified using the syntax
   description notation defined in [ABNF].

   date-fullyear   = 4DIGIT
   date-month      = 2DIGIT  ; 01-12
   date-mday       = 2DIGIT  ; 01-28, 01-29, 01-30, 01-31 based on
                             ; month/year
   time-hour       = 2DIGIT  ; 00-23
   time-minute     = 2DIGIT  ; 00-59
   time-second     = 2DIGIT  ; 00-58, 00-59, 00-60 based on leap second
                             ; rules
   time-secfrac    = "." 1*DIGIT
   time-numoffset  = ("+" / "-") time-hour ":" time-minute
   time-offset     = "Z" / time-numoffset

   partial-time    = time-hour ":" time-minute ":" time-second
                     [time-secfrac]
   full-date       = date-fullyear "-" date-month "-" date-mday
   full-time       = partial-time time-offset

   date-time       = full-date "T" full-time

      NOTE: Per [ABNF] and ISO8601, the "T" and "Z" characters in this
      syntax may alternatively be lower case "t" or "z" respectively.

      This date/time format may be used in some environments or contexts
      that distinguish between the upper- and lower-case letters 'A'-'Z'
      and 'a'-'z' (e.g. XML).  Specifications that use this format in
      such environments MAY further limit the date/time syntax so that
      the letters 'T' and 'Z' used in the date/time syntax must always
      be upper case.  Applications that generate this format SHOULD use
      upper case letters.

      NOTE: ISO 8601 defines date and time separated by "T".
      Applications using this syntax may choose, for the sake of
      readability, to specify a full-date and full-time separated by
      (say) a space character.
```

Reply:
👍  I think it looks cooler with the `T` and `Z`, but if that is what Datadog expects then it can't really be helped.